controllers/aquasecurity/aquastarboard/aquaStarboardHelper.go

```diff
@@ -65,141 +65,150 @@ func (enf *AquaStarboardHelper) CreateStarboardClusterRole(name string, namespac
 		},
 		{
 			APIGroups: []string{
-				"",
+				"apps",
 			},
 			Resources: []string{
-				"configmaps", "secrets", "serviceaccounts", "resourcequotas", "limitranges",
+				"replicasets", "statefulsets", "daemonsets", "deployments",
 			},
 			Verbs: []string{
-				"get", "list", "watch", "create", "update",
+				"get", "list", "watch",
 			},
 		},
 		{
 			APIGroups: []string{
-				"",
+				"batch",
 			},
 			Resources: []string{
-				"secrets",
+				"jobs", "cronjobs",
 			},
 			Verbs: []string{
-				"delete",
+				"get", "list", "watch",
 			},
 		},
 		{
 			APIGroups: []string{
-				"",
+				"rbac.authorization.k8s.io",
 			},
 			Resources: []string{
-				"events",
+				"roles", "rolebindings", "clusterroles", "clusterrolebindings",
 			},
 			Verbs: []string{
-				"create",
+				"get", "list", "watch",
 			},
 		},
 		{
 			APIGroups: []string{
-				"apps",
+				"apiextensions.k8s.io",
 			},
 			Resources: []string{
-				"replicasets", "statefulsets", "daemonsets", "deployments",
+				"customresourcedefinitions",
 			},
 			Verbs: []string{
 				"get", "list", "watch",
 			},
 		},
 		{
 			APIGroups: []string{
-				"batch",
+				"aquasecurity.github.io",
 			},
 			Resources: []string{
-				"jobs", "cronjobs",
+				"vulnerabilityreports", "configauditreports", "clusterconfigauditreports", "ciskubebenchreports",
 			},
 			Verbs: []string{
-				"get", "list", "watch",
+				"get", "list", "watch", "create", "update", "delete",
 			},
 		},
 		{
 			APIGroups: []string{
-				"rbac.authorization.k8s.io",
+				"coordination.k8s.io",
 			},
 			Resources: []string{
-				"roles", "rolebindings", "clusterroles", "clusterrolebindings",
+				"leases",
 			},
 			Verbs: []string{
-				"get", "list", "watch",
+				"create", "get", "update",
 			},
 		},
 		{
 			APIGroups: []string{
-				"apiextensions.k8s.io",
+				"networking.k8s.io",
 			},
 			Resources: []string{
-				"customresourcedefinitions",
+				"networkpolicies", "ingresses",
 			},
 			Verbs: []string{
 				"get", "list", "watch",
 			},
 		},
 		{
 			APIGroups: []string{
-				"batch",
+				"policy",
 			},
 			Resources: []string{
-				"jobs",
+				"podsecuritypolicies",
 			},
 			Verbs: []string{
-				"create", "delete",
+				"get", "list", "watch",
 			},
 		},
+	}
+
+	crole := rbac.CreateClusterRole(name, namespace, "starboard-operator", fmt.Sprintf("%s-rbac", "aqua-sb"), "Deploy Aqua Starboard Role", rules)
+
+	return crole
+}
+
+func (enf *AquaStarboardHelper) CreateStarboardRole(name string, namespace string) *rbacv1.Role {
+	rules := []rbacv1.PolicyRule{
 		{
 			APIGroups: []string{
-				"aquasecurity.github.io",
+				"",
```

Comment:
Why its empty?

Reply:
probably Goland removed it automatically for some reason,
couldn't delete those because I've done the changes accordingly to this PR and those should remain
https://github.com/aquasecurity/deployments/pull/488/files